### DISCUSSION TITLE
训练关键信息抽取模型（KIE）时，RE的关系怎么标注？

### DISCUSSION THREAD

**Discussion by Longleaves** [PaddlePaddle/PaddleOCR#12721] [Q&A]
根据官方文档（https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/ppstructure/kie/how_to_do_kie.md#22-%E8%AE%AD%E7%BB%83KIE%E6%A8%A1%E5%9E%8B）
我只找到SER的标注方式，使用命令：PPOCRLabel --lang ch --kie=True打开PPOCRLabel后，页面多了关键字列表：
![图片](https://github.com/PaddlePaddle/PaddleOCR/assets/58903935/9d017c7a-c95e-43f2-b918-761ffd4f4129)

也并没有文档中所说的other而是None，导出标签的字段为"key_cls": "None"。
准备进行下一步RE标注时发现不知从何入手，导出标签也没有label字段。官方文档实例图如下：
![184528728-626f77eb-fd9f-4709-a7dc-5411cc417dab](https://github.com/PaddlePaddle/PaddleOCR/assets/58903935/e872a4e6-ed5a-49cf-a556-bdc8ca8c3022)

想求教各位大佬与官方，怎么进行RE标注，实现关键词之间的关系？以及求教具体训练的方法。万分感谢！

**Comment by zzgHiker** [comment]
同问 +1

**Comment by Chenkeyi43** [comment]
自己修改成other，并且将 key_cls 改成 label

**Comment by dandelion915** [comment]
楼主搞明白了吗 我也想问

**Comment by SWHL** [comment]
因PPOCRLabel已经搬家了，后续有问题请移步[PPOCRLabel](https://github.com/PFCCLab/PPOCRLabel)提问哈！
此issue先行关闭了。
